### PR TITLE
Drop support under ruby 2.7

### PR DESCRIPTION
**Pull request by marocchino**

### CODE REVIEW

File: test/test_phone_number_sg.rb [hunk 1]
```diff
@@ -46,7 +46,7 @@ def test_premium_service_number
 
   def test_phone_number
     10.times do
-      assert_match(/[6,8,9]\d{3}\s\d{4}/, @tester.phone_number)
+      assert_match(/[6,89]\d{3}\s\d{4}/, @tester.phone_number)
```
Comment (AlexWayfer) [inline test/test_phone_number_sg.rb:49]: Is this intentional change?

**Review by AlexWayfer — COMMENTED** [review]
I'm currently working on RuboCop offenses (complete) resolution, and maybe even `lint` in CI. So it'd help me a lot.

File: test/test_phone_number_sg.rb (hunk 1 above)
Comment (AlexWayfer) [reply test/test_phone_number_sg.rb:49]: Oh, it's chars in Regexp, haven't noticed. Probably then we should move `,` somewhere else, before or after digits.

Also it looks strange: phone number begins with `,`? Probably there should not be `,` at all.

Comment (marocchino) [reply test/test_phone_number_sg.rb]: 2409bd026c894e777036415859c002e55ab10541

**Pull request merged**